rotkehlchen/tests/unit/decoders/test_curve.py

```diff
@@ -730,7 +731,7 @@ def test_curve_remove_imbalanced(database, ethereum_transaction_decoder):
             asset=A_USDC,
             balance=Balance(amount=FVal('665.267705'), usd_value=ZERO),
             location_label=location_label,
-            notes='Receive 665.267705 USDC from the curve pool 0xbBC81d23Ea2c3ec7e56D39296F0cbB648873a5d3',  # noqa: E501
+            notes='Remove 665.267705 USDC from the curve pool',
```

Comment:
yeah see? The information is there. Want it back. Why remove it?

Reply:
> yeah see? The information is there. Want it back. Why remove it?

For this test, the address passed isn't a pool. It's the zap contract(Curve Y Deposit). Added it back but with the correct metapool.